plugins/tracemgmt/api/resp.go

```diff
@@ -0,0 +1,18 @@
+package api
+
+const (
+	// TracingEnabled represents enabled state
+	TracingEnabled = "enabled"
+
+	// TracingDisabled represents disabled state
+	TracingDisabled = "disabled"
+)
+
+// JaegerConfigInfo represents tracing configration
+type JaegerConfigInfo struct {
+	Status               string  `json:"tracing-status"`
+	JaegerEndpoint       string  `json:"jaeger-endpoint"`
```

Comment:
can we embed `SetupTracingReq`  into `JaegerConfigInfo` i see configInfo contains the same fields

Reply:
I prefer leaving it the way it is for readability. Nesting will add another indirection to access members of the original structure and there are a lot of places where these members are accessed/modified.

Can I take this up as a separate change set? Please let me know.